python/tank/platform/events/event_file_close.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from .event_engine import EngineEvent
+
+class FileCloseEvent(EngineEvent):
+    """
+    An object representation of a file-open event.
+    """
+    def __init__(self):
+        """
+        Constructor.
+
+        :param str file_path: The path to the file opened.
```

Comment:
doc string seems to be cut and pasted from the file open event. 

Seems like the design of the file close event should send some sort of handle back to the user to indicate what file is actually being closed, so that MDI DCCs can correctly implement this. I think it's generally helpful if possible (even in SDI DCCs) for integrating code to pass the current file that is being closed.